src/style.css

```diff
@@ -1 +1,32 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars:first-child {
+  margin-top: 0;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url('./images/star.svg');
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-left: 4px;
+}
```

Comment:
You're using a fixed width and height for `.stars__star`, which is generally fine, but make sure this does not lead to content overflow or restrictions if the container size changes or if the content within the star elements needs more space. If the content might vary, consider using min-width and min-height instead.